.github/workflows/engine_test.yml

```diff
@@ -21,7 +23,7 @@ jobs:
         python-version:  ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python install.py devel
+        python install.py devel --version=$GITHUB_HEAD_REF
```

Comment:
Why this addition? Is it to get the requirements file updated in this branch?

Reply:
Yes, otherwise it always takes requirements from master